sections/futures/Trade/SmartMarginOnboardModal.tsx

```diff
@@ -0,0 +1,111 @@
+import dynamic from 'next/dynamic';
+import React, { memo } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import BaseModal from 'components/BaseModal';
+import Error from 'components/ErrorView';
+import { FlexDivRowCentered } from 'components/layout/flex';
+import Spacer from 'components/Spacer';
+import { selectSusdBalance } from 'state/balances/selectors';
+import { selectIsolatedTransferError } from 'state/futures/selectors';
+import { useAppSelector } from 'state/hooks';
+import { formatDollars } from 'utils/formatters/number';
+
+type Props = {
+	onDismiss(): void;
+};
+
+const SocketBridge = dynamic(() => import('../../../components/SocketBridge'), {
+	ssr: false,
+});
+
+const SmartMarginOnboardModal: React.FC<Props> = memo(({ onDismiss }) => {
+	const { t } = useTranslation();
+
+	const txError = useAppSelector(selectIsolatedTransferError);
```

Comment:
@LeifuChen this is a SmartMargin component but referencing isolatedTransfer error, is that expected?

Reply:
No. It's an irrelevant selector as we don't have any deposit and withdraw logic in this component.